Dockerfile

```diff
@@ -0,0 +1,18 @@
+#Using the official ubuntu OS as a Base Image
+FROM ubuntu:16.04
+
+USER root
+
+#Configuration
```

Comment:
These files will need to present in order to build the image. For the pip install, you could just do an ADD/COPY. However, I tend to think volumes might be better for this. I suggest two volumes- one for the e2e test infrastructure and one for the edx-platform repo. That way, you can easily symlink them to edx-e2e-tests/lib and run install_pages.